backend/app/rest/api/rest_private.go

```diff
@@ -221,6 +224,134 @@ func (s *private) voteCtrl(w http.ResponseWriter, r *http.Request) {
 	render.JSON(w, r, R.JSON{"id": comment.ID, "score": comment.Score})
 }
 
+// sendConfirmation gets address and siteID from query, makes confirmation token and sends it to user.
+// GET /email?site=site&address=someone@example.com
+func (s *private) sendConfirmation(w http.ResponseWriter, r *http.Request) {
+	if !s.emailNotificationsEnabled {
+		rest.SendErrorJSON(w, r, http.StatusNotImplemented, errors.New("unavailable"), "email notifications disabled", rest.ErrNotificationsDisabled)
+		return
+	}
+	user := rest.MustGetUserInfo(r)
+	address := r.URL.Query().Get("address")
+	siteID := r.URL.Query().Get("site")
+	if address == "" {
+		rest.SendErrorJSON(w, r, http.StatusBadRequest, errors.New("missing parameter"), "address parameter is required", rest.ErrInternal)
+		return
+	}
+	existingAddress, _ := s.dataService.GetUserDetail(store.Locator{SiteID: siteID}, user.ID, "email")
+	if address == existingAddress {
+		rest.SendErrorJSON(w, r, http.StatusBadRequest, errors.New("wrong request"), "address is already verified", rest.ErrInternal)
+		return
+	}
+	claims := token.Claims{
+		Handshake: &token.Handshake{
+			State: "",
+			ID:    user.ID + "::" + address,
+		},
+		StandardClaims: jwt.StandardClaims{
+			Audience:  r.URL.Query().Get("site"),
+			ExpiresAt: time.Now().Add(30 * time.Minute).Unix(),
+			NotBefore: time.Now().Add(-1 * time.Minute).Unix(),
+			Issuer:    "remark42",
+		},
+	}
+
+	tkn, err := s.authenticator.TokenService().Token(claims)
+	if err != nil {
+		rest.SendErrorJSON(w, r, http.StatusForbidden, err, "failed to make verification token", rest.ErrInternal)
+		return
+	}
+
+	////TODO: figure out how to send emails
```

Comment:
I can't figure out how to call `Email.SendVerification` from there, would be grateful for ideas.

Reply:
Addresed by https://github.com/paskal/remark/pull/2